systori/apps/project/api.py

```diff
@@ -105,6 +143,31 @@ def search(self, request):
 
         return JsonResponse({"projects": projects})
```

Comment:
see above, not sure what's that about. did you replace something existing or what is this?

Reply:
I didn't make any changes to the `search` method, if that is what you are referring to here.